src/device_update_import/env_vars.md

```diff
@@ -0,0 +1,70 @@
+# Environment variables
+This command uses an Azure Application Identity for its access to the Azure Device Update. It gets its inputs from environment variables.
+
+## Authorization needed to access the Azure Device Update is acquired by app identity
+AZURE_TENANT_ID=*azure-tenant-id*
+AZURE_CLIENT_ID=*azure-client-id*
+AZURE_CLIENT_SECRET=*azure-client-secret*
+AZURE_ACCOUNT_ENDPOINT="https://<device update resource>.api.adu.microsoft.com"
+
+## Instance id of the update instance inside of the device update resource referenced above
+AZURE_INSTANCE_ID=*azure-instance-id*
+
+## Azure Blob Storage where the update is stored
+AZURE_STORAGE_NAME=*azure-storage-name*
+AZURE_STORAGE_KEY=*azure-storage-key*
+AZURE_BLOB_CONTAINER=*azure-container-name*
+
+## Import manifest
+
+### Compatibility information
+ADU_DEVICEPROPERTIES_MANUFACTURER=*adu-manufacturer*
+ADU_DEVICEPROPERTIES_MODEL=*adu-model*
+ADU_DEVICEPROPERTIES_COMPATIBILITY_ID=*adu-compat-id*
+
+### Update provider
+ADU_PROVIDER=*adu-provider*
+
+### UpdateId
+OMNECT_IMAGE_NAME=*swupdate-image-name*
+
+### Version
+OMNECT_IMAGE_VERSION=*omnect-os-version*
+
+### Default step type of swupdate step
+ADU_UPDATE_TYPE=*swupdate-step-type* (should be `microsoft/swupdate:2`)
+
+### InstalledCriteria
+
+OMNECT_IMAGE_INSTALLED_CRITERIA="omnect-image-installed-criteria*br>
```

Comment:
remove "br>" and opening quotes

Reply:
wasn't fixed before merge

Reply:
forgot to commit. will be fixed in next release.